### ISSUE TITLE
Decoding FIP loss equation 

### ISSUE THREAD

**Issue by HariniNarasimhan** [Guruprasad93/FlexibleMachineLearning#3]
Hi @Guruprasad93 ,
Thanks for this solution to solve catastrophic forgetting and it is very interesting to learn. Please help me to decode the below which I find it difficult to understand.

`loss1 = torch.sum(-torch.log(torch.sum(torch.sqrt(output*output_ori+epsAdd), axis=1)))`

https://github.com/Guruprasad93/FlexibleMachineLearning/blob/8374c5a63576be176ab6d98dad2ec3b87a52f31c/VIT/train_fip.py#L223

The loss1 is computed but not used for the loss to back propagate. Can you please help me with reasoning?
